index.html

```diff
@@ -68,6 +68,128 @@ <h2>I'am a Software Developer</h2>
         </div>
         <div class="background"></div>
       </section>
+      <section class="works">
```

Comment:
There is an issue regarding horizontal scrolling in your portfolio that is damaging the user experience at the moment. As you can see in the following image, the content overflows the width of the screen and the user needs to scroll left of right to see the rest of it 😥

![image](https://user-images.githubusercontent.com/15807343/153536903-9a0ed0fa-c110-4ea4-86f0-511dfb488738.png)

Kindly make the necessary modifications to your code so that the issue is addressed 🧐

Reply:
I have tried to replicate the issue in both chrome and safari but cannot seem to get the same result. I am attaching a picture of a screen size of 375px and an iPhone 12 Pro
<img width="445" alt="375px" src="https://user-images.githubusercontent.com/45613824/153598080-f38d84e0-bb44-45c1-a467-8c9ce33c3b28.png">
<img width="446" alt="iphone12Pro" src="https://user-images.githubusercontent.com/45613824/153598091-0fd3e80b-aff7-439b-9db3-c39b099b0d05.png">
.

Reply:
What a weird issue 😵 I tried your project locally and using online tools and my browser displayed it with that issue! Maybe it was on my side! Hopefully other reviewers don't face the same issue 🚀 Best of luck!